### ISSUE TITLE
Save report to file

### ISSUE THREAD

**Issue by bregman-arie** [bregman-arie/reqwise#2]
Right now reqwise will print the report to the console. We should allow to keep the report in a file:

reqwise -o report.txt

**Comment by mramanathan** [comment]
@bregman-arie, Is this issue about capturing output from print statements and redirect them to output file as per users' choice ? 

If so, existing functionality (which I have not gone thru' fully yet), should be retained as well (i.e. if there are print statement redirecting the output to standard console) ?

**Comment by bregman-arie** [comment]
@mramanathan yes, if user specified '-o <output_file>' then the output of the logging lines (there is no use of print statements) should be also redirected to a file in addition to console output.

Thanks for the interest.

**Comment by ShanikaEdiriweera** [comment]
Hi @bregman-arie, I would like to tackle this issue with your guidance.

**Comment by ShanikaEdiriweera** [comment]
@bregman-arie I have done the fix. How do I run the modified code on local machine to test?

**Comment by ghost** [comment]
@bregman-arie  can i work on this?

**Comment by bregman-arie** [comment]
@ShanikaEdiriweera sorry for the late reply. Missed the notification in mail.
Run with 'reqwise -o report.txt' and see if the file created. If yes, send a pull request and I'll merge it :)

**Comment by bregman-arie** [comment]
@shreab373 sure! thank you :)

**Comment by ShanikaEdiriweera** [comment]
I still can run `reqwise -o report.txt` since issue #6 is still there.

**Comment by bregman-arie** [comment]
@ShanikaEdiriweera did you run pip install -r requirements.txt from the repo itself?

pypi didn't have latest version. I updated it now.
